hashicorp/boundary-ui
# Icu 12592 add filtering pagination support for auth methods

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12592

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-1292)

## Description


add search, filtering, and pagination support to auth methods

## Screenshots (if appropriate):
![image](https://github.com/hashicorp/boundary-ui/assets/107949262/40f6282a-ea5c-49d5-857a-c73648bf97fd)

## How to Test


1. start boundary dev instance
2. test search on auth methods
3. test type filtering on auth methods
4. test primary filtering on auth methods
5. Validate that filtering and searching works as expected


:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
